packages/localize-tools/package.json

```diff
@@ -22,7 +22,9 @@
     "clean": "git clean -dXn | sed 's/^Would remove //' | grep -v node_modules | xargs rm -rf",
     "build": "npm run clean && npm run generate-json-schema && tsc --build",
     "generate-json-schema": "typescript-json-schema tsconfig.schema.json ConfigFile --include=src/types/config.d.ts --required --noExtraProps > config.schema.json",
-    "test": "npm run test:unit && npm run test:check-tsc",
+    "test": "npm run test:dev && npm run test:prod",
+    "test:dev": "npm run test:unit && npm run test:check-tsc",
```

Comment:
It feels a little weird to have `test:dev` and `test:prod` in packages where that distinction doesn't have any meaning.

It also seems like it would be easy to mistakenly create a new package (like `cli`) which has a `test` target, and assume that will automatically get run during CI, because that is the widely used convention that most packages used, but with this scheme it won't get run at all.

So I would suggest a slight variation:

- Use `test` for `prod` tests, and for packages where there is no dev vs prod distinction.

- Use `test:dev` only in the packages where that distinction exists.

Also note that `lerna run <script>` only runs `<script>` in the packages that have that script, so we don't need to define no-op `test:dev` scripts in packages without the prod vs dev distinction.

Reply:
If there is no distinction or discernable difference between DEV and PROD then we should not be running two sets of tests.

If there is a functional and _testable_ difference between the patterns and behaviors of DEV and PROD mode, then we should separate these environments / contexts / whatever.

Isolating DEV failures and PROD failures is critical.

If we are incorrectly using `staging` terminology and the only difference is whether DEV logs more output than PROD, then we should only test PROD and leave DEV as a `--verbose` mode to help with tests.